### ISSUE TITLE
Unicode problem handling export status message

### ISSUE THREAD

**Issue by justinlittman** [gwu-libraries/sfm-ui#258]
Steps to reproduce:
1. Create an export request for a Weibo harvest that does not have any WARCs. The seed set name should contain unicode characters (e.g., 微博测试).
2. Wait.

Expected result:
Export request will be marked as completed failure and the user will receive an email notification.

Actual result:
Export request remains marked as requested and the user does not receive an email.

sfm_ui_consumer is crashing with:
```
Traceback (most recent call last):
  File "/opt/sfm-ui/sfm/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/__init__.py", line 346, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 394, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/dist-packages/django/core/management/base.py", line 445, in execute
    output = self.handle(*args, **options)
  File "/opt/sfm-ui/sfm/message_consumer/management/commands/startconsumer.py", line 20, in handle
    consumer.run()
  File "/usr/local/lib/python2.7/dist-packages/kombu/mixins.py", line 177, in run
    for _ in self.consume(limit=None):  # pragma: no cover
  File "/usr/local/lib/python2.7/dist-packages/kombu/mixins.py", line 199, in consume
    conn.drain_events(timeout=safety_interval)
  File "/usr/local/lib/python2.7/dist-packages/librabbitmq/__init__.py", line 220, in drain_events
    self._basic_recv(timeout)
  File "/usr/local/lib/python2.7/dist-packages/kombu/messaging.py", line 598, in _receive_callback
    return on_m(message) if on_m else self.receive(decoded, message)
  File "/usr/local/lib/python2.7/dist-packages/kombu/messaging.py", line 564, in receive
    [callback(body, message) for callback in callbacks]
  File "/usr/local/lib/python2.7/dist-packages/sfmutils/consumer.py", line 121, in _callback
    self.on_message()
  File "/opt/sfm-ui/sfm/message_consumer/sfm_ui_consumer.py", line 32, in on_message
    self._on_export_status_message()
  File "/opt/sfm-ui/sfm/message_consumer/sfm_ui_consumer.py", line 155, in _on_export_status_message
    seed_set.name, export_url)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-3: ordinal not in range(128)
```

**Comment by Tanych** [comment]
1. The weibo harvester supports seedless harvester. Refs PR https://github.com/gwu-libraries/sfm-weibo-harvester/pull/13
2. sfm-ui email message formating needs to support unicode. Refs PR https://github.com/gwu-libraries/sfm-ui/pull/260

**Comment by kerchner** [comment]
Resolved in #260
